src/DateRangePicker/test/DateRangePickerSpec.js

```diff
@@ -624,4 +624,23 @@ describe('DateRangePicker', () => {
 
     expect(getByLabelText('gear')).to.have.class('rs-icon');
   });
+
+  it('Should render a custom calendar title', () => {
+    const { getAllByTestId } = render(
+      <DateRangePicker
+        value={[new Date(2022, 1, 1), new Date(2022, 2, 2)]}
+        open
+        renderTitle={date => (
+          <span data-testid="calendar-title">
+            {date.getMonth()} - {date.getFullYear()}
+          </span>
+        )}
+      />
+    );
+
+    const [firstTitle, secondTitle] = getAllByTestId('calendar-title');
+
+    expect(firstTitle.innerText).to.equal('1 - 2022');
+    expect(secondTitle.innerText).to.equal('2 - 2022');
```

Comment:
Use `.textContent` instead of `.innerText` to retrieve don content. Or use `.to.have.text(value)` assertion directly.

Reply:
Hi @SevenOutman, just pushed the change requested. Thanks for the review!